internal/verify/test_fixture.go

```diff
@@ -96,7 +96,7 @@ func (tf *testFixture) runTestSuite(ctx context.Context, eng *engine.Engine, sho
 		}
 
 		testResult := TestResult{Name: TestName{TableTestName: test.Name.TestTableName, PrincipalKey: test.Name.PrincipalKey}}
-		if test.Skip || !shouldRun(test.Name.TestTableName) {
+		if test.Skip || !shouldRun(fmt.Sprintf("%s/%s", ts.Name, test.Name.String())) {
```

Comment:
Maybe we can run `shouldRun` check for both the test suite and each test in it.

Reply:
The reason I removed the check from the suite is because it makes it tricky to write a regex for an individual test. For example, if you write `AlbumTestSuite.*delete.*`, then the match would fail for the suite name and we would skip the whole suite -- which prevents that specific test we wanted from being executed as well.